docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+```
+
+- The value of `db_users` corresponds to the IAM role created in the previous step.
+- The value(s) for `db_names` will depend on your Redshift Serverless configuration,
+  but `dev` is the default name applied by AWS. You can also provide `*` to grant
+  access to all instances.
+
+Save this file and apply it to your Teleport cluster:
+
+```code
+$ tctl create -f redshift-role.yaml
+role 'redshift-serverless-access' has been created
+```
+
+Add this role to your Teleport user. You can do this through the web UI, or by
+downloading your user definition:
+
+```code
+$ tctl get users/yourUser > yourUser.yaml
+```
+
+Adding the role:
+
+```yaml
+...
+  roles:
+  - redshift-serverless-access
+...
+```
+
+And applying the changes:
+
+```code
+$ tctl create -f yourUser.yaml
+user "yourUser" has been updated
+```
+
+## Step 4/5. Install and start the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+On the same host, create a Teleport configuration file:
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:443 \
+   --token=/tmp/token \
+   --redshift-serverless-discovery=us-west-1
+```
+
+- Replace `teleport.example.com` with your Teleport proxy URI or cloud tenant
+  address (e.g. `mytenant.teleport.sh`).
+- Provide the token from the previous step to the `--token` flag.
+- Update the value of `--redshift-serverless-discovery` to match your AWS region.
+
+This command will generate a Database Service configuration with Redshift
+Serverless auto-discovery enabled and place it at `/etc/teleport.yaml`.
+
+<Admonition type="tip" title="Filter with AWS tags">
+Use `--aws-tags <key>=<value>,<key>=<value>` to only match those with specified
+AWS resource tags.
+
+For example, if a workgroup has multiple VPC endpoints but
+only one of the VPC endpoints will be accessed through Teleport, add the resource
+tag `teleport.dev/discovery` with value true to the VPC endpoint, and then specify
+`--aws-tags teleport.dev/discovery=true` when running `teleport db configure create`.
+</Admonition>
+
+### Bootstrap access to Redshift Serverless
+
+Teleport can bootstrap IAM permissions for the Database Service based on its
+configuration using the `teleport db configure bootstrap` command. You can use
+this command in automatic or manual mode:
+
+- In automatic mode, Teleport will attempt to create appropriate IAM policies
+  and attach them to the specified IAM identity (user or role). This requires the
+  IAM permissions we applied to the `teleport-redshift-node` role earlier.
+- In manual mode, Teleport will print required IAM policies. You can then create
+  and attach them manually using the AWS management console.
+
+<Admonition type="note" title="AWS credentials">
+AWS Credentials are only required if you’re running the command in "automatic"
```

Comment:
How do you supply AWS credentials? I only see that they are required. If the intention is to provide this information in the "Specifying Credentials" link, what information am I supposed to extract from the link?

Reply:
Clarifying in the next commit.